recipes/r-metarep/meta.yaml

```diff
@@ -0,0 +1,74 @@
+{% set version = '1.2.0' %}
+{% set posix = 'm2-' if win else '' %}
+
+package:
+  name: r-metarep
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/metarep_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/metarep/metarep_{{ version }}.tar.gz
+  sha256: c9ea8481e702f3236e74fb3b37b2816778a1b48349e190b05de0591f6cdbae13
+
+build:
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+  missing_dso_whitelist:
+    - '*/R.dll'        # [win]
+    - '*/Rblas.dll'    # [win]
+    - '*/Rlapack.dll'  # [win]
+
+requirements:
+  build:
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+  host:
+    - r-base
+    - r-meta >=6.0_0
+  run:
+    - r-base
+    - r-meta >=6.0_0
+
+test:
+  commands:
+    - $R -e "library('metarep')"           # [not win]
+    - "\"%R%\" -e \"library('metarep')\""  # [win]
+
+about:
+  home: https://github.com/IJaljuli/metarep
+  license: GPL-2.0-or-later
+  summary: |
+    User-friendly package for reporting replicability-analysis methods, affixed to meta-analyses
+    summary. The replicability-analysis output provides an assessment of the investigated
+    intervention, where it offers quantification of effect replicability and assessment
+    of the consistency of findings.
+  license_file:
+    - '{{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-2'
+
+extra:
+  recipe-maintainers:
+    - conda-forge/r
+
+# Package: metarep
+# Title: Replicability-Analysis Tools for Meta-Analysis
+# Version: 1.2.0
+# Depends: R (>= 4.1), meta (>= 6.0-0)
+# Suggests: metafor (>= 1.9.9), lme4, numDeriv, BiasedUrn, knitr, rmarkdown
+# Authors@R: person("Iman","Jaljuli", role=c("cre","aut"), email="jaljuli.iman@gmail.com")
+# Date: 2023-12-15
+# URL: https://github.com/IJaljuli/metarep
+# Description: User-friendly package for reporting replicability-analysis methods, affixed to meta-analyses summary. The replicability-analysis output provides an assessment of the investigated intervention, where it offers quantification of effect replicability and assessment of the consistency of findings. - Replicability-analysis for fixed-effects and random-effect meta analysis: - r(u)-value; - lower bounds on the number of studies with replicated positive and\or negative effect; - Allows detecting inconsistency of signals; - forest plots with the summary of replicability analysis results; - Allows Replicability-analysis with or without the common-effect assumption.
+# License: GPL (>= 2)
+# Encoding: UTF-8
+# NeedsCompilation: yes
```

Comment:
This is weird, but from looking at [the package code](https://github.com/cran/metarep/tree/1.2.0), there is indeed nothing compiled. So, I think the recipe is fine as is.